src/main/webapp/jsp/instructorFeedbackResultsTop.jsp

```diff
@@ -10,12 +10,14 @@
 <r:sessionPanel sessionPanel="${data.sessionPanel}" />
 
 <c:if test="${not (noResponses and data.allSectionsSelected and showAll)}">
-  <r:filterPanel filterPanel="${data.filterPanel}"
-      showAll="${showAll}" />
+  <r:filterPanel filterPanel="${data.filterPanel}"/>
 </c:if>
 <br>
 <t:statusMessage statusMessagesToUser="${data.statusMessagesToUser}" />
-<br>
+<c:if test="${not (noResponses and data.allSectionsSelected and showAll)}">
```

Comment:
@whipermr5 slight confused with naming the variable. How about  `testForNoResponsesAndShowAll` ?